_includes/context/priority.md

```diff
@@ -0,0 +1,11 @@
+
+There are two settings that determine determine the priority: `predicate` and `scope`. The priority system mirrors the old [inlite](https://www.tiny.cloud/docs/themes/inlite/#quicklink) plugin from TinyMCE 4. The `predicate` is a function that takes in the current context position and returns a boolean. The `scope` is either `node` or `editor`. The whole priority process works as follows:
```

Comment:
In TinyMCE4, inlite was a theme. In TinyMCE5, it's a plugin. All references to inlite from TinyMCE4 need to say theme.